### ISSUE TITLE
set max number of photos for selection?

### ISSUE THREAD

**Issue by rcholic** [hyperoslo/ImagePicker#76]
Thanks for the great work!
I have a couple of questions/problems:

1. I am wondering if there is any way to set the number of photos that a user can select using this `ImagePicker`.

2. XCode complains that `imagePicker` as an unresolved identifier in the following code:
```
public var imageAssets: [UIImage] {
        return ImagePicker.resolveAssets(imagePicker.stack.assets)
    }
```

3. How to dismiss the image picker when the user taps the "Done" button in on the photo selection UI? For now, nothing happens when I tap on "Done"

**Comment by RamonGilabert** [comment]
Hey @rcholic! :)

To do the maximum number of images thing that you say, you should point to master instead of using the pod from cocoapods, since we haven't released yet (But we will soon version 1.1). That was fixed, you can see the issue in #66.

To dismiss the ImagePicker, you call the delegate method and, with the instance of your variable:

dimissViewController(imagePickerController, animated: true), the default method with iOS.

We'll check the complain thing.

Thank you. :)

**Comment by rcholic** [comment]
thanks, but I actually found it in the `development` branch, not the `master`. Thanks a lot!
